Makefile

```
@@ -0,0 +1,59 @@
+.PHONY: install start start-dev lint test test-unit test-e2e pack dist dist-linux \
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)